Makefile

```diff
@@ -41,10 +41,12 @@ AUTHOR ?= $(shell git config user.email || echo $$USER)
 ENABLE_METRICS ?= true
 BUILD_FLAGS ?= $(shell echo "-ldflags='\
 	-X github.com/status-im/status-go/params.Version=$(RELEASE_TAG:v%=%) \
+	-X github.com/status-im/status-go/params.Mode=$(mode) \
```

Comment:
I tend to prefer a specific parameter, rather than a `mode` for the whole app, i.e
params.IPFSGatewayMode, or rather just pass the URL:
params.IPFSGatewayURL, but maybe @jakubgs has a preference for one or the other, I don't have a strong opinion on it, though catchall mode makes is less explicit imo.

Reply:
I agree that `Mode` is far too generic of a name for a paremeter.

Reply:
I think if we have to do it at build time `params.IPFSGatewayURL` makes more sense as it's more explicit on what it does an makes it easier for people to find and customize if they want to. `Mode` can be literally anything.

Reply:
I used this Mode way because I thought maybe in the future, we'd have more per build mode configurations, so that's why I left it kinda generic.